pass/pass.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const PASS_FOLDER = "docker-credential-helpers"
 
-// Pass handles secrets using Linux secret-service as a store.
+// Pass handles secrets using secret-service as a store.
```

Comment:
Actually wondering here if the description was correct for the `pass` credentials helper, or if this was a copy/paste error from the sercretservice helper? ("using secret-service as a store")

Reply:
Fair point. I admittedly had no idea what `secret-service` was when I was working on this, so I decided to take the lazy way out and leave most of the comment untouched.

How's "Pass handles secrets using GPG encrypted files as a store" sound?

Or maybe the comment is entirely useless, I'm not really sure.

Reply:
👍 for the comment change.

Note that `pass` is already described in the [package overview](https://pkg.go.dev/github.com/docker/docker-credential-helpers/pass), so the comment on that symbol is not that important.